.gitignore

```diff
@@ -21,3 +21,5 @@ dist
 *.sw?
 .cypress/screenshots/*
 .cypress/videos/*
+
+.env
```

Comment:
Could you add in this PR to the repo .env.template and cmd in the README to copy it.

Reply:
Ah this in fact belongs to my other PR, I'll move it there!